### ISSUE TITLE
Have an option to allow large uploads

### ISSUE THREAD

**Issue by arthurlutz** [YunoHost-Apps/redirect_ynh#4]
For a site I need to add 

```
  client_max_body_size 10G;
```

to the nginx options... how could this become an option ? or a generic mechanism to that an advanced user can add custom nginx rules or options ? Will the customized version of the config file be ovewritten at some point ?

**Comment by scith** [comment]
Hi,
This app sets a generic Nginx file that is supposed to work in most circumstances. Unfortunatenly, it cannot accommodate specific cases easily.

I suggest you try customize the file located at /etc/nginx/conf.d/YOURDOMAIN.d/YOURREDIRECT.conf
There is also a topic on the forum where you can share your customized nginx file, which could help other users too https://forum.yunohost.org/t/redirect-ynh-share-your-case-examples-and-config/2182

Would that work?
Thanks :+1:

**Comment by gamersalpha** [comment]
Hello
I for thé dame problème


Maybe Can add a option attend thé installation for this paramètre

Merci

**Comment by opi** [comment]
Good idea to have a "custom rule" option during installation. Anyone feel free to contribute, I don't have so much time for this app nowdays
